dice/cime_config/buildnml

```diff
@@ -11,15 +11,21 @@
 
 import os, sys
 
-_CIMEROOT = os.path.join(os.path.dirname(os.path.abspath(__file__)), "..","..","..","..","..")
-sys.path.append(os.path.join(_CIMEROOT, "scripts", "Tools"))
+_CDEPS_CONFIG = os.path.join(os.path.dirname(os.path.abspath(__file__)), os.pardir,os.pardir,"cime_config")
```

Comment:
see comments from datm

Reply:
We decide to stay with this at this point.